src/zone/zone.ts

```diff
@@ -104,9 +94,8 @@ export class Zone {
   }
   public set shrinkCoefficient(value: number) {
     if (value <= 0) {
-      throw new Error("Invalid shrinking coefficient. The value should be greater then 0.");
-    }
-    else {
+      throw new Error('Invalid shrinking coefficient. The value should be greater then 0.');
```

Comment:
did it mean, that if bot send incorrect data then game cycle will break?

Reply:
No. When you create zone instanse with wrong parameters, it will throw an error.

Reply:
Bot does not interact with the zone algorithm